{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2022-25024"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "The json2xml package through 3.12.0 for Python allows an error in typecode decoding enabling a remote attack that can lead to an exception, causing a denial of service."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://packaging.python.org/en/latest/guides/analyzing-pypi-package-downloads/",
          "refsource": "MISC",
          "tags": [],
          "url": "https://packaging.python.org/en/latest/guides/analyzing-pypi-package-downloads/"
        },
        {
          "name": "https://github.com/vinitkumar/json2xml/pull/107/files",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/vinitkumar/json2xml/pull/107/files"
        },
        {
          "name": "https://github.com/vinitkumar/json2xml/pull/107",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/vinitkumar/json2xml/pull/107"
        },
        {
          "name": "https://github.com/vinitkumar/json2xml/issues/106",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/vinitkumar/json2xml/issues/106"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2023-08-22T20:10Z",
  "publishedDate": "2023-08-22T19:16Z"
}